packages/server/src/routes/members/give.ts

```diff
@@ -196,10 +196,10 @@ const _createGiveOperationAndNotify = async (
       data: {
         to_uid: memberToGiveToId,
         amount: toAmount.toString(),
-        memo,
         donation_to: donationRecipient.id,
         donation_amount: donationAmount.toString(),
-        metadata
+        metadata,
+        ...(memo && [memo])
```

Comment:
What's the goal of this spread operator here? `...[memo]` will result in `{[0]: <memo>}`, whereas `...{memo}` will result in `{memo: <memo>}` which I think is what you want?